packages/builtin/string.pony

```diff
@@ -880,12 +880,14 @@ actor Main
 
       n = 0
 
-      while n < len do
+      while (n < len) or (len == USize(-1)) do
```

Comment:
what's the len == USize(-1) case?

Reply:
nvm, i went and looked at the surrounding code. given that len == -1 is the default, i suspect the ordering should be reversed. if -1 is the most common, we should test for that first.

Reply:
ah right. sorry. sick and not quite with it today.